lib/libc/aarch64/Symbol.map

```diff
@@ -28,7 +28,12 @@ FBSD_1.0 {
 	makecontext;
 };
 
+FBSD_1.3 {
```

Comment:
With the symbol moved here, the reason for FBSD_1.3 (vs FBSD_1.0) version is less obvious (the reason is that it matches the non-emutls symbol that is over in locale/Symbol.map).  I think perhaps for that reason I probably prefer leaving it over in locale.